Modules/Filtering/Convolution/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-project(ITK-Convolution)
-
+project(ITKConvolution)
+set(ITKConvolution_LIBRARIES ITKConvolution)
```

Comment:
This should probably be removed, as Convolution is a header-only module.

Reply:
This will be needed when the next PR with the enum classes is added.  The enum classes print function needs to be in a library.   I'm just trying to split compound patch sets apart into simpler pieces.

Reply:
I will submit just the 1 patch with the isolated FFT library fix, and then move the ITKConvolution patch back to the enum-c PR.